course_content/cartopy_course/cartopy_intro.ipynb

```diff
@@ -87,26 +81,56 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's compare the Plate Carree projection to another projection from the projection list; being the Miller projection. We'll do that by plotting two subplots next to each other at the same time:"
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Suggested Activity :</font></b> Try modifying the above code to plot a different projection.\n",
+    "</div>"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#\n",
+    "# EDIT for user code ...\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION :  Un-comment and execute the following to see a possible solution ...\n",
+    "# (which you can also then run)\n",
+    "\n",
+    "# %load solutions/cartopy_exercise_1"
```

Comment:
This is inconsistent with the Iris course. In the Iris course The explanation bit is in markdown

Reply:
Only for the first one, but yes if you think that was clearer, I'll make this the same.